LibertyDSNP/spec
# favicon added, logo added, link to github added

Problem
=======
I want to see appropriate branding on the spec
So I know I'm within the Unfinished ecosystem
[link to Pivotal Tracker #12345678](https://www.pivotaltracker.com/story/show/12345678)

Solution
========
* The top left corner of the site should include the unfinished logo instead of the words "Your Logo"
* The Github link in the top right should go to GitHub.com/LibertyDSNP/spec
* Update the favicon with the Unfinished logo
* Do not include any styling, which is captured in #176772238

Change summary:
---------------
* add unfinished logo
* add unfinshed logo favicon
* add link to correct github repo (not public yet so link gets a 401, but goes to the correct address)

Steps to Verify:
----------------
1. run site and check that you see the correct logo and favicon
2. press github link to check it

Screenshots (optional):
-----------------------
![Screen Shot 2021-02-16 at 4 55 04 PM](https://user-images.githubusercontent.com/43625033/108141042-d4e30900-7077-11eb-8232-5a6bbdfdcf3d.png)
